cicecore/cicedynB/dynamics/ice_dyn_shared.F90

```diff
@@ -1563,7 +1563,7 @@ subroutine strain_rates_T (nx_block,   ny_block, &
       DeltaT = sqrt(divT**2 + e_factor*(tensionT**2 + shearT**2))
 
     end subroutine strain_rates_T
-      
```

Comment:
Let's maybe avoid unnecessarily changing whitespace here

Reply:
I can't find the changed whitespace...let's forget about this.

Reply:
You can just git checkout the file to revert to the original, then commit it again.